src/master-playlist-controller.js

```diff
@@ -430,7 +403,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
       if (!updatedPlaylist) {
         // select the initial variant
-        this.initialMedia_ = this.selectPlaylist();
+        const initialPlaylistSelector =
+          this.enableLowInitialPlaylist ?
+            this.selectInitialPlaylist : this.selectPlaylist;
+
+        this.initialMedia_ = initialPlaylistSelector();
```

Comment:
You can do this without needing to create the `initialPlaylistSelector` variable